src/gluelayer.d

```diff
@@ -55,14 +55,11 @@ version (NoBackend)
 }
 else
 {
-    import ddmd.backend.type;
+    public import ddmd.backend.cc : block, Blockx, Symbol;
```

Comment:
The backend imports are still a work in progress. I'd prefer dealing with them in a later PR, with this one I just got things fixed enough to compile tocsym.d and pass the test suite.